src/ppom-controller.ts

```diff
@@ -97,13 +98,16 @@ export type PPOMState = {
   storageMetadata: FileMetadataList;
   // ETag obtained using HEAD request on version file
   versionFileETag?: string;
+  // Where the downloaded files are stored
+  fileStorage: Record<string, string>;
 };
 
 const stateMetaData = {
   versionInfo: { persist: false, anonymous: false },
   chainStatus: { persist: false, anonymous: false },
   storageMetadata: { persist: false, anonymous: false },
   versionFileETag: { persist: false, anonymous: false },
+  fileStorage: { persist: true, anonymous: false },
 };
```

Comment:
I think it will be useful to also make `chainStatus` persistent and we are moving files to controller state.